packages/amazon-sumerian-hosts-babylon/test/integration_test/README.md

```diff
@@ -1,8 +1,17 @@
-# Running Integration/Stress Test
+# Integration Tests
 
-Each integration test contains an end to end example focusing on a single Host feature like animation, text-to-speech etc. 
+Each integration test contains an end to end example focusing on a single Host feature like animation, text-to-speech etc. These tests run in the browser and must be exercised manually.
 
-## Walkthrough
+## Prerequisites
 
-1. Replace any ```<Enter Cognito Identity Pool ID here>``` with your own AWS IdentityPoolId or create your own AWS credential if you're testing texttospeech feature. Follow tutorial [here](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/getting-started-browser.html) if you need to set up your own cognito identity pool.
-2. Run ```npm run start-babylon``` from the repository root to launch the web server and navigate to the corresponding test page.
\ No newline at end of file
+In order for the integration tests to be runnable you will need to set up a few things in your AWS account. For step-by-step instructions on setting up this required infrastructure, see [AWS-Infrastructure-Setup.md](../../../../AWS-Infrastructure-Setup.md) in the root of this repository.
+
+## Setup
```

Comment:
Is this still accurate? I thought we moved this to `demo-credentials.js`? I think this applies to all of the packages.

Reply:
Thanks for catching this. I will fix.